src/main/resources/config/application-dev.yml

```diff
@@ -106,6 +106,10 @@ jhipster:
 
 # Properties to be exposed on the /info management endpoint
 info:
+    tutorial-course-groups: # Names of the tutorial groups that will be automatically added to newly created users
```

Comment:
I suggest we put this into the guided-tour section below where all other properties are configured

Reply:
you could then name it `course-group-students`, etc.